src/layouts/HomeCards/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { describe, test, expect, vi, Mock } from 'vitest';
+import { render } from '@testing-library/react';
+import { HomeCards } from '.';
+import { fetchData, fetchedData } from '../../utils/requests';
+import { LoadingIcon } from '../../assets/LoadingIcon';
+import { Provider } from 'react-redux';
+import store from '../../store/store';
+
+global.fetch = vi.fn(() =>
+  Promise.resolve({
+    json: () => Promise.resolve(fetchedData),
+  })
+) as Mock;
```

Comment:
Overriding globals is not great, but at least you should restore them after your test.